lib/export_task.rb

```diff
@@ -24,6 +24,13 @@ def export(output_dir, options)
     @exporter.system_description.assert_scopes("os")
     @exporter.system_description.validate_export_compatibility
 
+    ["unmanaged_files", "changed_managed_files", "config_files"].each do |scope|
+      if @exporter.system_description[scope] &&
+        !@exporter.system_description.scope_extracted?(scope)
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.